File: sakila-payment-service/src/main/java/com/example/sakila/application/SakilaPaymentService.java
package com.example.sakila.application;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ComponentScan;

@SpringBootApplication
@EnableAutoConfiguration
@ComponentScan("com.example.sakila")
public class SakilaPaymentService {

  public static void main(String[] args) {
    SpringApplication.run(SakilaPaymentService.class, args);
  }
}

File: sakila-payment-service/src/main/java/com/example/sakila/application/SakilaPaymentService.kt
package com.example.sakila.application

import org.springframework.boot.runApplication
import org.springframework.boot.autoconfigure.EnableAutoConfiguration
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient
import org.springframework.context.annotation.ComponentScan

@SpringBootApplication
@EnableDiscoveryClient
@EnableAutoConfiguration
@ComponentScan("com.example.sakila")
class SakilaPaymentService

fun main(args: Array<String>) {
  runApplication<SakilaPaymentService>(*args)
}
